include/geometric_shapes/bodies.h

```diff
@@ -74,30 +74,24 @@ struct BoundingCylinder
   EIGEN_MAKE_ALIGNED_OPERATOR_NEW
 };
 
-// to be able to use the more efficient Eigen::Transform::linear() instead of .rotation(), we need to check the user has
-// really passed an isometry; to avoid runtime costs, this check is only done as assert, which get compiled-out in
-// release builds
+// To be able to use the more efficient Eigen::Transform::linear() instead of rotation(),
+// we need to check the user has really passed an isometry. To avoid runtime costs,
+// this check is only done as assert, which get compiled-out in release builds
 #ifdef NDEBUG
 #define ASSERT_ISOMETRY(transform) assert(true);
 #else
-#define ASSERT_ISOMETRY(transform)                                                                                     \
-  {                                                                                                                    \
-    bool isIsometry = (transform).matrix()(3, 0) == 0 && (transform).matrix()(3, 1) == 0 &&                            \
```

Comment:
@rhaschke Are you sure this check is done by design? Wouldn't that only be the case in the "efficient storage" which only stores a 3x4 matrix?

Reply:
True. Will revert.